app/main.py

```diff
@@ -4,4 +1,30 @@
+import requests
+import os
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
Checklist point 2: Use uppercase style for constants' names. You did well with `BASE_URL`, but remember to apply this rule to all your constants.